grails-app/controllers/org/broadinstitute/orsp/api/NewConsentGroupController.groovy

```diff
@@ -127,4 +127,36 @@ class NewConsentGroupController extends AuthenticatedController {
         response.status = 200
         render([consent: consent] as JSON)
     }
+
+    Collection<ConsentCollectionLink> getConsentCollectionLinks() {
+        Issue issue = queryService.findByKey(params.consentKey)
+        Collection<ConsentCollectionLink> collectionLinks = queryService.findCollectionLinksByConsentKey(issue.projectKey)
+        render([collectionLinks: collectionLinks.linkedProject] as JSON)
+        collectionLinks
+    }
+
+    /**
+     * This action breaks the link between a project and the consent from the point of view of the consent.
+     *
+     */
+    def unlinkAssociatedProjects () {
+        Issue issue = queryService.findByKey(params.consentKey)
```

Comment:
really small naming nit here and in `getConsentCollectionLinks`. Since this is finding a consent group, it should be named as such
```suggestion
        Issue consent = queryService.findByKey(params.consentKey)
```